src/McpResponse.ts

```diff
@@ -32,8 +38,39 @@ export class McpResponse implements Response {
     this.#includeSnapshot = value;
   }
 
-  setIncludeNetworkRequests(value: boolean): void {
+  setIncludeNetworkRequests(
+    value: boolean,
+    options?: NetworkRequestsListingOptions,
+  ): void {
     this.#includeNetworkRequests = value;
+    if (!value) {
+      this.#networkRequestsPaginationOptions = undefined;
+      return;
+    }
+
+    if (!options) {
+      this.#networkRequestsPaginationOptions = undefined;
+      return;
+    }
+
+    const sanitizedOptions: NetworkRequestsListingOptions = {};
```

Comment:
We do not need to sanitize the values as we can rely on the types and JSON schema.

Could we replace this with the following?

```
this.#networkRequestsPaginationOptions = options;
```